include/freerdp/crypto/rdptls.h

```diff
@@ -99,28 +99,28 @@ extern "C"
 {
 #endif
 
-	FREERDP_API const SSL_METHOD* tls_get_ssl_method(BOOL isDtls, BOOL isClient);
+	FREERDP_API const SSL_METHOD* rdp_tls_get_ssl_method(BOOL isDtls, BOOL isClient);
```

Comment:
since this is a public header, better rename to `freerdp_tls_*`, we mostly use the `rdp_*` prefix for non API functions